schemas/ns10_cache_entry.fbs

```diff
@@ -0,0 +1,13 @@
+
+file_identifier "ns10";
+
+
+/// pylint: skip-file
```

Comment:
I've suggested using `flake8` rather than `pylint`, based on Mantid's experiences using each tool, but `pylint` is also ok. If you have an opinion on this or other python static analysis/testing tools feel free to edit/comment on this wiki page:
https://confluence.esss.lu.se/display/DMSC/Testing+Strategy

Reply:
`pylint` is used by Jenkins in NICOS review system, and this compels us to use it.